common/src/main/java/org/wso2/testgrid/common/TestGridConstants.java

```diff
@@ -53,11 +54,15 @@
 
     public static final String DEFAULT_DEPLOYMENT_PATTERN_NAME = "default";
     public static final String TESTGRID_CONFIG_FILE = "config.properties";
+    public static final String TESTGRID_SCENARIO_OUTPUT_PROPERTY_FILE = "output.properties";
 
     public static final String WUM_USERNAME_PROPERTY = "WUMUsername";
     public static final String WUM_PASSWORD_PROPERTY = "WUMPassword";
 
     public static final String TEST_TYPE_FUNCTIONAL = "FUNCTIONAL";
     public static final String TEST_TYPE_PERFORMANCE = "PERFORMANCE";
 
+    public static final String TEST_PLANS_URI = "test-plans";
```

Comment:
Shall we use existing PRODUCT_TEST_PLANS_DIR ?

Reply:
Although both values are same, Test-plans-URI refers to the "test-plans" phrase in our TestGrid dashboard URLs, while Product-test-plans-Dir refers to the name of the directory where test-plans exists. Those two could change in future independently :) WDYT?